### PR TITLE
[Area 1] Adds screenshots for inline deployment

### PR DESCRIPTION
**Pull request by marciocloudflare**
Addresses PCX-5074

### CODE REVIEW

File: content/email-security/deployment/inline/setup/gsuite-area1-mx.md [hunk 1]
```diff
@@ -27,17 +27,31 @@ When Area 1 is deployed as MX records upstream of Gmail, the Inbound gateways ne
 
 2. Navigate to **Apps** > **Google Workspace** > **Gmail**.
 
+    ![Access Gmail](/email-security/static/inline-setup/gsuite-area1-mx/step2-gmail.png)
+
 3. Select **Spam, Phishing, and Malware** and scroll to **Inbound Gateway configuration**.
 
-4. Enable **Inbound Gateway**, and configure it with the following details: 
+    ![Access the spam and phishing setting](/email-security/static/inline-setup/gsuite-area1-mx/step3-spam.png)
```
Comment (angelampcosta) [inline content/email-security/deployment/inline/setup/gsuite-area1-mx.md]:
```suggestion
    ![Access the spam, phishing and malware setting](/email-security/static/inline-setup/gsuite-area1-mx/step3-spam.png)
```

File: content/email-security/deployment/inline/setup/gsuite-area1-mx.md [hunk 2]
```diff
@@ -53,27 +67,45 @@ If you would like to send Area 1 malicious detection to a separate quarantine ot
...
 3. In the quarantine configuration pop-up, enter the following:
 
     * **Name**: `Area 1 Malicious`.
     * **Description**: `Area 1 Malicious`.
     * For the **Inbound denial consequence**, select **Drop Message**.
     * For the **Outbound denial consequence**, select **Drop Message**.
 
+    <div class="medium-img">
+
+    ![Select the add quarantine button](/email-security/static/inline-setup/gsuite-area1-mx/step3-configure-quarantine.png)
```
Comment (angelampcosta) [inline content/email-security/deployment/inline/setup/gsuite-area1-mx.md]: This one has the some title as the one above. Is this correct?

File: content/email-security/deployment/inline/setup/gsuite-area1-mx.md [hunk 3]
```diff
@@ -87,10 +119,18 @@ When you are finished entering these details, select **SAVE**.
...
+
+    ![Configure the compliance filter](/email-security/static/inline-setup/gsuite-area1-mx/step3-compliance-filter.png)
+
+    </div>
+
     After you enter this information, select **SAVE**.
 
 4. Once saved, the console will update with the newly configured **content compliance filter**.
 
+    ![After configuratiuon, the console shows the content compliance filter](/email-security/static/inline-setup/gsuite-area1-mx/step4-compliance-filter.png)
```
Comment (angelampcosta) [inline content/email-security/deployment/inline/setup/gsuite-area1-mx.md]:
```suggestion
    ![After configuration, the console shows the content compliance filter](/email-security/static/inline-setup/gsuite-area1-mx/step4-compliance-filter.png)
```

File: content/email-security/deployment/inline/setup/office-365-area1-mx.md [hunk 4]
```diff
@@ -19,12 +19,20 @@ In this tutorial, you will learn how to configure Microsoft Office 365 with Area
...
 
+    ![Select the anti-spam option](/email-security/static/inline-setup/o365-area1-mx/step3-anti-spam.png)
+
 4. Select **Connection filter policy (Default)** to edit the policy, and then select **Edit connection filter policy**.
 
+    ![Select edit connection filter policy](/email-security/static/inline-setup/o365-area1-mx/step4-edit-filter-policy.png)
+
 5. In **Always allow messages from the following IP addresses or address range** add the IP addresses and CIDR blocks mentioned in [Egress IPs](/email-security/deployment/inline/reference/egress-ips/).
 
+    ![Enter the egress IP addressess](/email-security/static/inline-setup/o365-area1-mx/step5-egress-ips.png)
```
Comment (angelampcosta) [inline content/email-security/deployment/inline/setup/office-365-area1-mx.md]:
```suggestion
    ![Enter the egress IP addresses](/email-security/static/inline-setup/o365-area1-mx/step5-egress-ips.png)
```

File: content/email-security/deployment/inline/setup/office-365-area1-mx.md [hunk 5]
```diff
@@ -86,32 +104,76 @@ This option will allow Office 365 to properly identify the original connecting I
 
 1. From the **Microsoft Exchange admin center**, select **mail flow** > **connectors**.
 
+    ![Select connectos from mail flow](/email-security/static/inline-setup/o365-area1-mx/step1-connectors.png)
```
Comment (angelampcosta) [inline content/email-security/deployment/inline/setup/office-365-area1-mx.md]:
```suggestion
    ![Select connectors from mail flow](/email-security/static/inline-setup/o365-area1-mx/step1-connectors.png)
```

File: content/email-security/deployment/inline/setup/office-365-area1-mx.md [hunk 6]
```diff
@@ -86,32 +104,76 @@ This option will allow Office 365 to properly identify the original connecting I
...
+
+    ![Provide name and description for the new connector](/email-security/static/inline-setup/o365-area1-mx/step3-description.png)
+
+    </div>
+
 4. Select **Use the sender’s IP address** in the **How do you want to identify the partner organization?** configuration panel. Select **Next**.
 
+    <div class="large-img">
+
+    ![Select the senders IP address](/email-security/static/inline-setup/o365-area1-mx/step4-senders-ip.png)
```
Comment (angelampcosta) [inline content/email-security/deployment/inline/setup/office-365-area1-mx.md]:
```suggestion
    ![Select the sender's IP address](/email-security/static/inline-setup/o365-area1-mx/step4-senders-ip.png)
```

File: content/email-security/deployment/inline/setup/office-365-area1-mx.md [hunk 7]
```diff
@@ -86,32 +104,76 @@ This option will allow Office 365 to properly identify the original connecting I
...
+
+    </div>
+
 5. In **What sender IP addresses do you want to use to identify your partner?** add the IP addresses and CIDR blocks found in [Egress IPs](/email-security/deployment/inline/reference/egress-ips/). Select **Next**.
 
+    <div class="large-img">
+
+    ![Enter Area 1's egress IP addressess](/email-security/static/inline-setup/o365-area1-mx/step5-office-egress-ips.png)
```
Comment (angelampcosta) [inline content/email-security/deployment/inline/setup/office-365-area1-mx.md]:
```suggestion
    ![Enter Area 1's egress IP addresses](/email-security/static/inline-setup/o365-area1-mx/step5-office-egress-ips.png)
```

File: content/email-security/deployment/inline/setup/cisco-area1-mx.md [hunk 8]
```diff
@@ -32,6 +32,7 @@ To add a new Sender Group:
 
 4. Select **Submit and Add Senders** and add the IP addresses mentioned in [Egress IPs](/email-security/deployment/inline/reference/egress-ips/).
 
+![Sender group.](/email-security/static/inline-setup/cisco-area1-mx/step1.png)
```
Comment (angelampcosta) [inline content/email-security/deployment/inline/setup/cisco-area1-mx.md]:
```suggestion
![Sender group](/email-security/static/inline-setup/cisco-area1-mx/step1.png)
```

File: content/email-security/deployment/inline/setup/cisco-cisco-mx.md [hunk 9]
```diff
@@ -32,6 +32,8 @@ To add a new Sender Group:
 
 4. Select **Submit and Add Senders** and add the IP addresses mentioned in [Egress IPs](/email-security/deployment/inline/reference/egress-ips/).
 
+![Sender group.](/email-security/static/inline-setup/cisco-cisco-mx/step1.png)
```
Comment (angelampcosta) [inline content/email-security/deployment/inline/setup/cisco-cisco-mx.md]:
```suggestion
![Sender group](/email-security/static/inline-setup/cisco-cisco-mx/step1.png)
```

Comment (angelampcosta) [reply content/email-security/deployment/inline/setup/cisco-cisco-mx.md]: These were the only two with punctuation.

File: content/email-security/deployment/inline/setup/office-365-area1-mx.md (hunk 5 above)
Comment (angelampcosta) [reply content/email-security/deployment/inline/setup/office-365-area1-mx.md]: mail vs email - but I guess in the UI says mail.

**Review by angelampcosta — APPROVED** [review]

File: content/email-security/deployment/inline/setup/gsuite-area1-mx.md (hunk 2 above)
Comment (marciocloudflare) [reply content/email-security/deployment/inline/setup/gsuite-area1-mx.md]: Corrected!

**Pull request merged**